src/qutip_qip/circuit_latex.py

```diff
@@ -240,7 +276,10 @@ def image_from_latex(code, file_type="png"):
                     raise ValueError(
                         "".join(["Unknown output format: '", file_type, "'."])
                     )
-                out = CONVERTERS[file_type](filename)
+                if file_type == "png" and dpi != 100:
+                    out = convert_to_img_with_dpi(filename, file_type, dpi)
+                else:
+                    out = CONVERTERS[file_type](filename)
```

Comment:
Is it possible to completely replace the old converter with this one? It is better to have as fewer duplications as possible. I feel like your `convert_to_img_with_dpi` can already almost replace the only converter. Maybe it can just read from the configurations and change the dpi setting?

Reply:
I haven't tried `convert_to_img_with_dpi` for SVG image. It's functionally possible to replace `_make_converter` with the new implementation. The advantage of having `_make_converter` function is that we don't have to run `_find_system_command` for every conversion like we do with `convert_to_img_with_dpi` and the converter functions are ready to use with fixed arguments.

Reply:
It could be quite often (refreshing the same circuit?) but the generation of the plot through `pdflatex` definitely takes much more time than `_find_system_command`, which checks if the command exists in the current system.

Even if we want to avoid running `_find_system_command` at each run, we can still put your converter inside `_make_converter`, right? So that the generated converter can take dpi as input (and probably raise an error if the dpi is given when the type is SVG).

Reply:
`_make_converter`_ takes density as argument now. Thanks